### ISSUE TITLE
Отображать имя модератора в подсказке удалённого тега

### ISSUE THREAD

**Issue by iDolmatov** [stalkerg/anime-pictures#96]
Предлагаю писать имя модератора, который удалил тег с опубликованной картинки.
Сейчас отображается "... удалено 204183":
1. Непонятное число 204183;
2. Непонятно кто ответственен за удаление тега, чтобы обсудить по необходимости, т.к. модератор выставления текущего статуса может быть другим.

**Comment by iDolmatov** [comment]
Если имя модератора нельзя писать, то 204183 заменить понятным словом/предложением.

**Comment by MiTereKun** [comment]
да можно написать имя, раз пишется id модератора. Сделают
`Кто ответственен за удаление тега<...>модератор выставления текущего статуса может быть другим.`
Ну это редкость, а после добавления счётчика "очереди проверки" - думаю случаев будет ещё меньше. Только если после проверки...

**Comment by iDolmatov** [comment]
Согласен, что второй тезис весьма редко применим, больше для веса предложения.

Недавно обновляли перевод, а by всё ещё не переведено для тегов картинок.

> Corrected tags for latest 20 pictures: | 137
Многовато выглядит, а подумаешь, что это и добавленные, то 7 тегов на картинку и не так уж много.

**Comment by stalkerg** [comment]
Увы пока так сделано сознательно т.к. тогда запрос к БД получается слишком тяжелым. 
Но я посмотрю что можно сделать... в целом это не такая частая необходимость узнать конкретно кто удалил тег так что ИМХО с этим пока жить можно.
